src/Vision/VNFaceLandmarkRegion2D.cs

```diff
@@ -0,0 +1,40 @@
+﻿//
+// VNFaceLandmarkRegion2D.cs
+//
+// Authors:
+//	Alex Soto  <alexsoto@microsoft.com>
+//
+// Copyright 2017 Xamarin Inc. All rights reserved.
+//
+
+#if XAMCORE_2_0
+
+using System;
+using Vector2 = global::OpenTK.Vector2;
+
+namespace XamCore.Vision {
+	public partial class VNFaceLandmarkRegion2D {
+
+		public Vector2 this [nuint index] {
+			get { return GetPoint (index); }
+		}
+
+		public Vector2 [] Points {
```

Comment:
@spouliot The method which this property calls is the one that is Virtual, do we also want to make this wrapper virtual?